common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4356,11 +4373,12 @@ object CelebornConf extends Logging {
   val DYNAMIC_CONFIG_STORE_BACKEND: ConfigEntry[String] =
     buildConf("celeborn.dynamicConfig.store.backend")
       .categories("master", "worker")
-      .doc("Store backend for dynamic config. Available options: NONE, FS. Note: NONE means disabling dynamic config store.")
+      .doc("Store backend for dynamic config service. Available options: NONE, FS, DB. Note: NONE means disabling " +
```

Comment:
Could we describe mort details for `FS` and `DB` to tell users which options should configure after the `celeborn.dynamicConfig.store.backend` is `FS` or `DB`?

Reply:
> Could we describe mort details for `FS` and `DB` to tell users which options should configure after the `celeborn.dynamicConfig.store.backend` is `FS` or `DB`?

I will add a doc for this feature later.

Reply:
IMO the current configurations may confusing users.

Both `NONE` and `FS` indicates reading conf from file, NONE does not support refresh but FS does. In such cases, would it be more clear to replace with the following configurations:
- celeborn.configuration.provider: <FILE|DB> 
- celeborn.configuration.refresh.interval: <None means disable|Some(int) means enable>

Reply:
Another concerns is only a few set of the configurations supports changed dynamiclly, where can user get such list?

Reply:
> Another concern is only a few subset of the configurations supports changing dynamiclly, where can user get such list?

Later I will add a isDynamic properties to celebornConf,  You can refer  https://issues.apache.org/jira/browse/CELEBORN-1051

Reply:
> IMO the current configurations may confusing users.
> 
> Both `NONE` and `FS` indicates reading conf from file, NONE does not support refresh but FS does. In such cases, would it be more clear to replace it with the following configurations:
> 
> * celeborn.configuration.provider: <FILE|DB>
> * celeborn.configuration.refresh.interval: <None means disabled|Some(int) means enabled>

Thank you @pan3793.  The None Option already removed.